src/com/janek/basics/math/MathUtil.java

```diff
@@ -29,4 +29,28 @@ public static double multiplyNumbers(double element1, double element2) {
     public static double divideNumbers(double dividend, double divider) {
         return dividend / divider;
     }
+
+
+    /**
+     * Calculates numbers
+     *
+     * @param one    first number
+     * @param symbol mathematical operation (+ - * /)
+     * @param two    second number
+     * @return calculated double number
+     */
+    public static double doCalculation(double one, String symbol, double two) {
+        switch (symbol) {
+            case "+":
```

Comment:
Nice method. But please introduce `public static final` constants for symbols `+_*/`
And then please use them in your main method ;)